resume.page

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Meryl Streep- Top 10</title>
+  <link rel="stylesheet" href="stylesheet.css" type="text/css" media="screen">
+  <nav>
+    <a href="index.html">Bio |</a>
+    <a href="resume.page">Greatest Hits |</a>
+    <a href="http://www.imdb.com/name/nm0000658/">More Info |</a>
```

Comment:
The common convention for external links is to open them up in a new tab/window.  Please fix.  🔨